### ISSUE TITLE
[Accessibility]A11y_Java Tooling for IntelliJ/Eclipse - New Features_ Spark Local Console – Locally run _Keyboard: In Light theme, Focus is not visible on “Help” button.

### ISSUE THREAD

**Issue by v-lasiva** [microsoft/azure-tools-for-java#3605]
[Check out Accessibility Insights! ](url)- Identify accessibility bugs before check-in and make bug fixing faster and easier.

GitHubTags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_JavaToolingforIntelliJ/Eclipse–New Feature; #MAS1.4.3; #MAS2.4.7; #Keyboard; #C+E-H; #JavatoolingforIntelliJ/Eclipse;

**Environment Details:**
URL: https://www.jetbrains.com/idea/download/#section=windows
Version: 2019.2.2
Build: 192.6603.28
MAS Violated: MAS 2.4.7

**Repro Steps:**
1.Download IntelliJ application from the URL:  https://www.jetbrains.com/idea/download/#section=windows .
2.Launch IntelliJ IDEA in Light theme.
3.Tab till Create New Project button and press enter
4.Tab till Azure Spark/HDInsight-->Spark Project Scala
5.Tab till Build Tools-->Maven
6.Tab till Next button and press enter
7.Tab till finish button and press enter
8. Navigate to Run --> Edit Configurations-->Apache Spark on HDInsight > [Spark on HDInsight] myApp
9.Tab till Locally run tab and press enter
10.Tab till “Enable parallel execution” check-box.
11.Tab once and verify whether focus is visible on “Help” button or not.

**Actual Result:**
In Light theme, Focus is not visible on “Help” button.

**Expected Result:**
In Light theme also, Focus should be visible on “Help” button.

**User Impact:**
Low vision users will face difficulty to know whether the focus is pointing on that “Help” button and will be unable to access them if focus is not visible in light theme.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={3b98b04f-4fab-4d3e-81af-a0906a34849c}

[Bug_6_Focus is not visible on Help button in Light theme in Spark Console.pptx](https://github.com/microsoft/azure-tools-for-java/files/3645423/Bug_6_Focus.is.not.visible.on.Help.button.in.Light.theme.in.Spark.Console.pptx)

**Comment by t-rufang** [comment]
Note: In the light theme, It's hard to find whether focus is at help button. This is supported by IntelliJ
![image](https://user-images.githubusercontent.com/32627233/65659979-d1547980-e05f-11e9-88cd-908d26d2be00.png)

**Comment by Alexandria-R** [comment]
This has been approved to close by SME.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749707
